lessons/Algebra2/exploring-linear-models/langs/en-us/pages/model-college-v-income.adoc

```diff
@@ -0,0 +1,67 @@
+= Fitting a Model: College Degrees v. Income
+++++
+<style>
+.studentAnswerMedium { min-width: 8em !important; }
+td { padding: 0 !important; }
+body.workbookpage .fitbruby { padding-top: 1.1rem; }
+body:not(.LessonPlan, .narrativepage) p { line-height: 1.2; }
+.center { padding: 0; }
+</style>
+++++
+
+For this page, you'll need to the @starter-file{alg2-states} open on your computer. If you haven't already, select **Save a Copy** from the "File" menu to make a copy of the file that's just for you.
+
+== Build a Model from Samples
+
+**Make a scatter plot showing the relationship between `college-or-higher` and `median-household-income`.**
+
+@n Record the `college-or-higher` and `median-household-income` values for Alabama (AL) and Alaska (AK), as (x,y) pairs below:
+
+@center{
+ (@fitbruby{10em}{@ifsoln{22.6}}{AK college-or-higher}, @fitbruby{10em}{@ifsoln{43253}}{AK median-household-income}) @hspace{10em} (@fitbruby{10em}{@ifsoln{27.5}}{AL college-or-higher}, @fitbruby{12em}{@ifsoln{70760}}{AL median-household-income})
+}
+
```

Comment:
I think this scaffolding is really helpful!
Is there a shortcut to help students find Alaska and Alabama on the scatterplot? I looked and looked before finding either one...

Reply:
They don't need to find them on the scatterplot! That's why they are defined as Rows in the starter , and explicitly evaluated and interpreted in `exploring-states.adoc`

Reply:
In that case, I would shift the directions and remind them to get the information from the table. 

Write now the ask for them to record the (x,y) pairs immediately follows the ask to build the scatter plot...

Still curious whether there's a way to find them more efficiently on the plot, because the fact that you're asking me to consider the line that connects them made me want to know where they were.